app/scripts/migrations/080.ts

```diff
@@ -0,0 +1,38 @@
+import { cloneDeep } from 'lodash';
+import { hasProperty, isObject } from '@metamask/utils';
+
+export const version = 80;
+
+/**
+ * The`@metamask/phishing-controller` state was updated in v2.0.0.
+ *
+ * @param originalVersionedData - Versioned MetaMask extension state, exactly what we persist to dist.
+ * @param originalVersionedData.meta - State metadata.
+ * @param originalVersionedData.meta.version - The current state version.
+ * @param originalVersionedData.data - The persisted MetaMask state, keyed by controller.
+ * @returns Updated versioned MetaMask extension state.
+ */
+export async function migrate(originalVersionedData: {
+  meta: { version: number };
+  data: Record<string, unknown>;
+}) {
+  const versionedData = cloneDeep(originalVersionedData);
+  versionedData.meta.version = version;
+  versionedData.data = transformState(versionedData.data);
+  return versionedData;
+}
+
+function transformState(state: Record<string, unknown>) {
+  if (
```

Comment:
Only out of curiosity: any reason to prefer this style to something like
```
if (!state.PhishingController) {
return state;
}
```

Reply:
and could `state.PhishingController` be anything other than an object?

Reply:
Yes; TypeScript doesn't like it when you reference properties that might not exist.

And for the `isObject` check it's the same; TypeScript doesn't like it when you destructure things that may-or-may-not be objects.

Reply:
okay, makes sense, thanks!

Reply:
>could state.PhishingController be anything other than an object?

I set the type here to `unknown` because it's coming from disk, so in theory it could be anything. I would never expect it to be anything other than an object in practice.